src/utils/constants.ts

```diff
@@ -68,3 +68,17 @@ export const formatUserObject = (userObject: UserEntity): GetUserDto => {
       : null,
   };
 };
+
+export const getUserData = (createUserResponse, partnerDetails, updatePartnerAccessResponse) => {
```

Comment:
Is this `getUserData` specifically for crisp? if so let's name it so as `getCrispUserData`

I think this function and the `getPartnerDetails` + `formatUserObject` functions shouldn't be in `constants.ts` as they're not constant enums? they could maybe go in a new `utils/serialise`